nss-evening-cohort-26/FE-Hackathon-music-streaming-project
# Add and Delete Song from Playlist

## Description
When a user clicks the view button (view details) of the Playlists, they should be able to see all the songs in their playlist if any. The user will be able to delete a song from the playlist via the delete button on the song. If the user has no songs in their playlist, they should see a button that says Add Song that allows them to add songs from a variety of available songs. A song that is in a user's playlist should not show in the variety of songs to add to the playlist.

## Related Issue
#10 

## Motivation and Context
The change is required to allow a user to add and remove songs from their created playlists.

## How Can This Be Tested?

- Click the eye button on the Playlist to see songs in the playlist. 
- If there are songs in the playlist, press delete to remove song from playlist and it will not show in the playlist anymore.
- If there are no songs in the playlist, click add song and it will show all available songs.
- Click the add button on the song that you would like to add to the playlist and you will get a pop up that the song was successfully added.
- To go back to the playlist, press the Back to Playlist button above all available songs.

## Screenshots (if appropriate):

## Types of changes
